docs/reference/installation.rst

```diff
@@ -25,42 +26,43 @@ Enable the Bundle
 
 .. code-block:: bash
 
-    php composer.phar require sonata-project/user-bundle --no-update
-    php composer.phar require sonata-project/doctrine-orm-admin-bundle  --no-update # optional
-    php composer.phar require friendsofsymfony/rest-bundle  --no-update # optional when using api
-    php composer.phar require nelmio/api-doc-bundle  --no-update # optional when using api
-    php composer.phar require sonata-project/google-authenticator --no-update  # optional
-    php composer.phar update
+    php composer require sonata-project/user-bundle --no-update
+    php composer require sonata-project/doctrine-orm-admin-bundle  --no-update # optional
+    php composer require friendsofsymfony/rest-bundle  --no-update # optional when using api
+    php composer require nelmio/api-doc-bundle  --no-update # optional when using api
+    php composer require sonata-project/google-authenticator --no-update  # optional
+    php composer update
 
-Next, be sure to enable the bundles in your and ``AppKernel.php`` file:
+Next, be sure to enable the bundles in your ``bundles.php`` file if they
```

Comment:
shouldn't we explain for sf 3 and sf 4?

on Sf 2/3 it should be AppKernel

Reply:
As flex is now the default way of creating new projects I though that should only be documented or am I wrong? @greg0ire I remember we talked about this on slack WDYT?

Reply:
We MUST assume everyone is using sf 4 and flex. We CAN add sf 2/3 instructions.

Reply:
That means I would need to add a note or something like `but if you are using symfony 3, do it this way`? so SHOULD I?

Reply:
It's really up to you. Personally, I think people that don't know how to register a bundle and also don't start with sf 4 might be quite rare, so it's probably not needed.

Reply:
I agree with that because current Symfony doc points to 4.0 and if you are using 3.4 you probably went with flex, if not then you know what you are doing. I do not mind adding docs for 3. If rest of the people think it should be added.